instance/static/html/instance/index.html

```diff
@@ -10,7 +10,7 @@
   <!-- Instances List -->
   <div class="large-3 columns">
     <ul class="side-nav side-instance-list">
-      <li ng-repeat="instance in instanceList | filter: {is_shut_down: false}"
+      <li ng-repeat="instance in instanceList | filter: {is_archived: false}"
```

Comment:
I considered that but decided against it for now because I think it's better to have a more flexible API, and we may have some other API consumer in the future that wants to be able to know about archived instances.

On second thought, now that `is_archived` is a database table, my preference would be to modify the query used by our frontend UI to append `?is_archived=true` to the query string (or make that the default, but preserve the possibility of using the API to list archived instances, just as you can use the API to get the details of an archived instance if you want to).